### ISSUE TITLE
Is it possible to add 2 software systems, their containers and their relationships on 1 view

### ISSUE THREAD

**Issue by kjellmoens** [structurizr/java#160]
Hi, 

For the moment, I got this but i can not add the containers to the diagram

  Model model = workspace.getModel();

        Person approver = model.addPerson(Location.Internal, "Approver", "");

        SoftwareSystem isilon = model.addSoftwareSystem(Location.Internal, "Isilon FTP", "");
        Container isilonCoda = isilon.addContainer("/coda", "", "Directory");
        Container isilonData = isilon.addContainer("/data", "", "Directory");
        Container isilonSepa = isilon.addContainer("/sepa", "", "Directory");

        SoftwareSystem sterling = model.addSoftwareSystem(Location.Internal, "Sterlin FTP", "");
        Container sterlingCoda = sterling.addContainer("/In/coda", "", "Directory");
        Container sterlingNotif = sterling.addContainer("/In/notif", "", "Directory");
        Container sterlingSepa  = sterling.addContainer("/Out/sepa", "", "Directory");

        SoftwareSystem btm = model.getSoftwareSystemWithName("BTM");
        Container coda = btm.addContainer("Coda", "", "Java, Spring");
        Container sepa = btm.addContainer("Sepa", "", "Java, Spring");
        Container database = btm.addContainer("Database", "", "FTP");
        database.addTags(DATABASE_TAG);

        ////////////////////////////////////////////////////////////////////////////////////////////////////////////////
        approver.uses(sepa, "approves");

        coda.uses(database, "");
        coda.uses(isilonCoda, "puts");
        coda.uses(isilonData, "puts");
        sepa.uses(isilonSepa, "gets");

        sepa.uses(database, "");
        coda.uses(sterlingCoda, "gets");
        coda.uses(sterlingNotif, "gets");
        sepa.uses(sterlingSepa, "puts");

        // define some views (the diagrams you would like to see)
        ViewSet views = workspace.getViews();

        ContainerView containerView = views.createContainerView(btm, "Containers", "The container diagram for the BTM System.");
       containerView.add(approver);
        containerView.add(isilon);
        containerView.add(sterling);
        containerView.add(isilonCoda);
        containerView.add(isilonData);
        containerView.add(isilonSepa);
        containerView.addAllContainersAndInfluencers();
        containerView.addAllContainers();
        containerView.addDependentSoftwareSystems();

        // add some documentation
        StructurizrDocumentationTemplate template = new StructurizrDocumentationTemplate(workspace);
        template.addContainersSection(btm, Format.Markdown,
                "Here is some context about the software system...\n" +
                        "\n" +
                        "![](embed:SystemContext)");

        return containerView;

**Comment by kjellmoens** [comment]
Hi Simon,

Thank you for the swift response .... 

Can I ask with what you rendered them ? I used the plantUml writer.
Also is it possible to have the 3 containers enclosed in the Isilon Software system?

**Comment by simonbrowndotje** [comment]
Yes, you can do this with the structurizr.com renderer, by adding:

```
containerView.setExternalSoftwareSystemBoundariesVisible(true);
```

![Screenshot 2021-03-18 at 15 44 21](https://user-images.githubusercontent.com/1009874/111654708-e9462d00-8800-11eb-8d79-5d3d499ecb9b.png)

It doesn't look like the PlantUML renderers support this though. I can transfer the issue to the `java-extensions` project if you like.

**Comment by kjellmoens** [comment]
Hi Simon,

Yes please, I also noticed why my diagram is not complete.

![image](https://user-images.githubusercontent.com/8494608/111655373-e3a11500-8809-11eb-8ab2-33a43a3cfd14.png)

The plantuml does not contain the labels for 48 - 50.

rectangle "==Approver\n<size:10>[Person]</size>" <<46>> as 46
rectangle "==Isilon FTP\n<size:10>[Software System]</size>" <<47>> as 47
rectangle "==Payback\n<size:10>[Software System]</size>" <<13>> as 13
rectangle "==Sterlin FTP\n<size:10>[Software System]</size>" <<51>> as 51
rectangle "==first\n<size:10>[Software System]</size>" <<4>> as 4
package "BTM\n[Software System]" {
  rectangle "==Coda\n<size:10>[Container: Java, Spring]</size>" <<55>> as 55
  rectangle "==Database\n<size:10>[Container: FTP]</size>" <<57>> as 57
  rectangle "==Sepa\n<size:10>[Container: Java, Spring]</size>" <<56>> as 56
}
46 .[#707070].> 56 : "approves"
55 .[#707070].> 48 : "puts"
55 .[#707070].> 49 : "puts"
55 .[#707070].> 57 : ""
56 .[#707070].> 50 : "gets"
56 .[#707070].> 57 : ""
